modules/ballerina-lang/src/main/java/org/wso2/ballerinalang/programfile/ProgramFileWriter.java

```diff
@@ -303,15 +303,6 @@ private static void writeConnectorInfo(DataOutputStream dataOutStream,
         // TODO write property flags  e.g. public
         dataOutStream.writeInt(connectorInfo.signatureCPIndex);
 
-//        Map<Integer, Integer> methodTable = connectorInfo.getMethodTableIndex();
```

Comment:
Let's get rid of comments :)

Reply:
it's already removed in my local and synced with this branch, not sure how it appears here :(